Sources/Fuzzilli/FuzzIL/TypeSystem.swift

```diff
@@ -906,8 +906,8 @@ extension Type: ProtobufConvertible {
     init(from proto: ProtoType) throws {
         var ext: TypeExtension? = nil
         
-        if !proto.properties.isEmpty || !proto.methods.isEmpty || proto.hasGroup || proto.hasSignature {
-            ext = TypeExtension(group: proto.hasGroup ? proto.group : nil,
+        if !proto.properties.isEmpty || !proto.methods.isEmpty || proto.group.isEmpty || proto.hasSignature {
```

Comment:
I think this should be `!proto.group.isEmpty`?

Reply:
That's a silly error on my part. Should be good to go!